src/pet.js

```diff
@@ -0,0 +1,79 @@
+function Pet(petName) {
+    const initialFitness = 10;
+    this.name = petName;
+    this.age = 0;
+    this.hunger = 0;
+    this.fitness = initialFitness;
+    this.children = [];
+};
+
+Pet.prototype = {
+    get isAlive() {
+    const unfit = 0;
+    const starving = 10;
+    const tooOld = 30;
+    return this.age < tooOld && this.hunger < starving && this.fitness > unfit;
+}
+};
```

Comment:
The indentation is out here, since you are inside a function there should be indentation to make it easy to see quickly. Assuming you dont refactor for the constants above this should look like this:
```
Pet.prototype = {
    get isAlive() {
        const unfit = 0;
        const starving = 10;
        const tooOld = 30;

        return this.age < tooOld && this.hunger < starving && this.fitness > unfit;
    }
};
```
Its a common convention to leave a clear line before the `return` statement also